decoders/a7105/__init__.py

```diff
@@ -0,0 +1,25 @@
+##
+## This file is part of the libsigrokdecode project.
+##
+## Copyright (C) 2014 Jens Steinhauser <jens.steinhauser@gmail.com>
```

Comment:
this doesn't look right, and the file has ..... nothing in it anyway, perhaps just drop the claim altogether?